src/main/java/com/questhelper/quests/bigchompybirdhunting/BigChompyBirdHunting.java

```diff
@@ -294,7 +296,8 @@ public void setupSteps()
 
 		killChompy = new NpcStep(this, NpcID.CHOMPY_BIRD, new WorldPoint(2635, 2966, 0), "Kill the chompy. You can only hurt it with an ogre bow + ogre arrows.", ogreBow, ogreArrows);
 		pluckCarcass = new NpcStep(this, NpcID.CHOMPY_BIRD_1476, new WorldPoint(2635, 2966, 0), "Pluck the chompy.");
-		talkToRantzWithChompy = new NpcStep(this, NpcID.RANTZ, new WorldPoint(2631, 2982, 0), "Talk to Rantz in the east of Feldip Hills.", chompy);
+		talkToRantzWithChompy = new NpcStep(this, NpcID.RANTZ, new WorldPoint(2631, 2982, 0), "Pick up the chompy and use it on Rantz in the east of Feldip Hills.", chompyHighlighted);
```

Comment:
I imagine this was less an issue of talking to him not working, and more holding spacebar causing the check for what ingredient he wants being skipped. If using it on him makes it such that a spacebar being held down doesn't fall on the skippable points, that works though.

Reply:
I also tested this myself yesterday while fixing it, and even when not holding spacebar it still caused it to be skipped. I did verify that using it works on him. Hoping to test the whole quest again soon to make sure that it still advances properly.